src/acbm/validating/utils.py

```diff
@@ -94,6 +98,11 @@ def calculate_od_distances(
     projected_epsg: int
         EPSG code for the projected CRS (default is 3857). We need a metric crs
         to calculte distances in meters.
+    detour_factor: float
+        Factor to adjust the estimated distance.
+    decay_rate: float
```

Comment:
As discussed, for the decay rate, two things that might be worth adding to config docs:
- Add description around the decay rate being the inverse of the distance (in units of the data, e.g. metres) at which the scaling from using the detour factor to Euclidean distance reduces by $\exp(-1)$
- Add link with [interactive plot](https://www.wolframalpha.com/input?i=plot+exp%28-0.0001x%29+from+x%3D0+to+x%3D50000) for exploring setting the decay rate

Reply:
Added the above comments in [22c98de](https://github.com/Urban-Analytics-Technology-Platform/acbm/pull/80/commits/22c98de7d3e9f3c8b31af707d875d722b68a2ae8)